.travis.yml

```diff
@@ -8,21 +8,21 @@ cache:
     - node_modules
 install: yarn --frozen-lockfile --non-interactive || (echo 'package.json is not in sync with yarn.lock, check that you include yarn.lock' && false)
 script:
-- yarn  
-- yarn lint  
-- yarn build 
+- yarn
+- yarn lint
+- yarn build
 - yarn test --forceExit --maxWorkers=4
 - yarn deploy && curl -sSL https://raw.githubusercontent.com/RedHatInsights/insights-frontend-builder-common/master/src/bootstrap.sh | bash -s
 env:
  global:
- - REPO="git@github.com:RedHatInsights/custom-policies-ui-frontend-build"
- - REPO_DIR="custom-policies-ui-frontend-build"
+ - REPO="git@github.com:RedHatInsights/policies-ui-frontend-build"
```

Comment:
@ryelo We are renaming from `custom-policies` to `policies`.

Could we get our *-build branch renamed to make them consistent ?

Reply:
Is this really something we need to update across the board? Changing the repo names won't affect anything on the actual UI, that's all handled in cloud services config

Reply:
We want to keep it consistent on every project / system to make everything easier to find.

Reply:
Sure thing. We just need to coordinate it with the cloud services config pull. I'm not sure, but I may have to regen the ssh key, so there might be other PRs I need to make here.